sql/core/src/test/resources/sql-tests/results/subquery/negative-cases/invalid-correlation.sql.out

```diff
@@ -70,9 +70,10 @@ Resolved attribute(s) t2b#x missing from min(t2a)#x,t2c#x in operator !Filter t2
 SELECT t1a 
 FROM   t1
 GROUP  BY 1
-HAVING EXISTS (SELECT 1 
+HAVING EXISTS (SELECT t2a
```

Comment:
why this change?

Reply:
@cloud-fan because, the child subquery plan can not be resolved due to missing attributes. 
```
+- Project [1#81]
:     +- !Filter (t2a#9 < min((outer(t1a#6) + t2a#9))#86)
:        +- Aggregate [1], [1 AS 1#81, min((outer(t1a#6) + t2a#9)) AS min((outer(t1a#6) + t2a#9))#86]
:           +- SubqueryAlias `t2`
:              +- Project [t2a#9, t2b#10, t2c#11]
:                 +- SubqueryAlias `t2`
:                    +- LocalRelation [t2a#9, t2b#10, t2c#11]
````

Reply:
But we do expect analysis exception for this query, don't we?

Reply:
@cloud-fan Yeah.. this particular test is in subquery suite and checking for a specific analysis expression that disallows a combination of local and outer references inside an aggregate expression. However, this test had the aggregate expression in the Filter clause which we are disallowing now. So i changed it to use "having" instead of "filter". But when i did that i hit a different analysis exception due to missing attributes (this particular test in question is trying to test a different analysis exception). So i fixed the projection here to include the missing attribute so we get a desired subquery related exception.

Reply:
makes sense